Remotion/Web/UnitTests/Core/ExecutionEngine/WxeContextTest.cs

```diff
@@ -275,6 +282,15 @@ public void GetPermanentUrlWithExistingReturnUrl ()
         () => _currentWxeContext.GetPermanentUrl(_functionType, queryString, true),
         Throws.ArgumentException);
   }
+
+  private ServiceLocatorScope CreateWxeUrlSettingsScope (int maxLength = 1024)
```

Comment:
```suggestion
  private ServiceLocatorScope CreateWxeUrlSettingsScopeWithDefaultWxeHandler (int maxLength = 1024)
```
otherwise, it's not obvious.

Also, I think we can the maxLength default as null? Alterantive: allow passing the defaultWxeHandler, too, and make the default as null. that way, only tests needed, have it set.